src/feeds/views.py

```diff
@@ -1616,3 +1623,26 @@ def check_match_query(request, user):
                 else:
                     return False
     return True
+
+
+def extract_tags(feed):
+    tag_index = []
+    delimiter_string = string.punctuation.translate(str.maketrans({'#':'', '_':''})) + string.whitespace
+    feed_list = re.split('([' + delimiter_string + '])', feed)
+    feed_list = [i for i in feed_list if i != '']
+    for i in range(len(feed_list)):
+        if len(feed_list[i]) > 1 and feed_list[i][0] == '#' and feed_list[i][1] != '_' and not '#' in feed_list[i][1:] and not feed_list[i].translate(str.maketrans({'#':'', '_':''})).isdecimal():
+            tag_index.append(i)
+    return feed_list, tag_index
+
+
+def create_link_tags(feed_list, tag_index):
```

Comment:
remove debug prints.

Reply:
Do you mean this https://github.com/s-tip/stip-sns/pull/95/commits/64fe07f7c353a85b84556a63b6349e26d9306acc. ?

Reply:
Yes. Thank you.